File: assignment-26/qustion2.cpp
#include<iostream>
using namespace std;

 class String
 {
    private:
    char *str;
    int iSize;
    public:
    String(){
        iSize=50;
        str=new char[iSize];
    }
    String(int x){
        int iSize=x;
        str=new char[iSize];
    }
    ~String(){
        delete []str;
    }
    void Accept(){
        cout<<"Enter string:\n";
        cin.getline(str,iSize);
    }
    int CountSmall(){
        int iCount=0;
        while(*str!='\0'){
            if(*str>='a' && *str<='z'){
                iCount++;
            }
            *str++;
        }
        return iCount;
    }
 };

int main(){
    int iRet=0;
    String *obj=new String();
    obj->Accept();
    iRet=obj->CountSmall();
    cout<<"count of small character are:"<<iRet;

   return 0;
}

File: assignment-32/qustion2.java
import java.util.*;

class StringDemo{
    public String str=null;

    public StringDemo(String x){
        this.str=x;
    }

    public int CountSmall(){
        char Arr[]=str.toCharArray();
        int iCnt=0;
        int iCount=0;
        for(iCnt=0;iCnt<Arr.length;iCnt++){
            if(Arr[iCnt]>='a' && Arr[iCnt]<='z'){
                iCount++;
            }
        }
        return iCount++;
    }
}
class qustion2{
    public static void main(String args[]){
        Scanner obj=new Scanner(System.in);
        int iRet=0;
        System.out.println("Enter the string:");
        String sobj=obj.nextLine();

        StringDemo cobj=new StringDemo(sobj);

        iRet=cobj.CountSmall();
        System.out.println("Number of small element: "+iRet);
    }
}